update-readmes.sh

```diff
@@ -28,7 +28,6 @@ strongloop loopback-connector-openapi master
 strongloop loopback-connector-oracle master
 strongloop loopback-oracle-installer master
 strongloop loopback-connector-postgresql master
-strongloop loopback-connector-remote master
```

Comment:
Will this affect LB3 docs https://loopback.io/doc/en/lb3/Remote-connector.html ?

Reply:
Taking a closer look at the script, it actually will affect LB3 readme too. Let me remove it. Thanks.